proj/src/main/webapp/static/chat.html

```diff
@@ -162,6 +162,21 @@
                     <a href="#" class="bg-light bg-grey list-group-item list-group-item-action">
                         <span class="menu-collapsed">change password</span>
                     </a>
+
+                    <a href="#account-submenu" data-toggle="collapse" aria-expanded="false" class="list-group-item list-group-item-action flex-column align-items-start">
+                            <span class="menu-collapsed">delete account</span>
+                    </a>
+                    <div id='account-submenu' class="collapse sidebar-submenu">
```

Comment:
The mixing of single and double quotes hurts me.

Reply:
sorry mat, never meant to hurt you; went back and changed all single quotes to double quotes